Source/SwiftLintFramework/Rules/CustomRules.swift

```diff
@@ -50,15 +69,19 @@ struct CustomRules: Rule, CacheDescriptionProvider, ConditionallySourceKitFree {
         name: "Custom Rules",
         description: """
             Create custom rules by providing a regex string. Optionally specify what syntax kinds to match against, \
-            the severity level, and what message to display.
+            the severity level, and what message to display. Rules default to SwiftSyntax mode for improved \
+            performance. Use `mode: sourcekit` or `default_execution_mode: sourcekit` for SourceKit mode.
```

Comment:
If there's `default_execution_mode`, should we use `execution_mode` instead of `mode`?